gensrc/proto/olap_file.proto

```diff
@@ -90,6 +90,8 @@ enum PartialUpdateMode {
     ROW_MODE = 1;
     COLUMN_MODE = 2;
     AUTO_MODE = 3;
+    // update only. not upsert
+    COLUMN_UPDATE_ONLY_MODE = 4;
```

Comment:
What's the difference between `COLUMN_UPDATE_ONLY_MODE` and `COLUMN_MODE`?

Reply:
COLUMN_UPDATE_ONLY_MODE = update
COLUMN_MODE = upsert = update + insert